src/python/WMCore/Services/AlertManager/AlertManagerAPI.py

```diff
@@ -0,0 +1,94 @@
+"""
+
+AlertManagerAPI - send alerts to MONIT AlertManager via API calls
+"""
+
+from __future__ import division
+from datetime import timedelta, datetime
+import socket
+import json
+import logging
+
+from WMCore.Services.pycurl_manager import RequestHandler
+from Utils.Timers import LocalTimezone
+
+
+class AlertManagerAPI(object):
+    """
+    A class used to send alerts via the MONIT AlertManager API
+    """
+
+    def __init__(self, alertManagerUrl):
+        self.alertManagerUrl = alertManagerUrl
+        # sender's hostname is added as an annotation
+        self.hostname = socket.gethostname()
+        self.mgr = RequestHandler()
+        self.ltz = LocalTimezone()
+
+    def send(self, alertName, severity, summary, description, tag="wmcore", service="", endSecs=600, generatorURL=""):
+        """
+        :param alertName: a unique name for the alert
+        :param severity: low, medium, high
+        :param summary: a short description of the alert
+        :param description: a longer informational message with details about the alert
+        :param service: the name of the service firing an alert
+        :param endSecs: how many minutes until the alarm is silenced
+        :param generatorURL: this URL will be sent to AlertManager and configured as a clickable "Source" link in the web interface
+
+        AlertManager JSON format reference: https://www.prometheus.io/docs/alerting/latest/clients/
+        [
+          {
+            "labels": {
+            "alertname": "<requiredAlertName>",
+            "<labelname>": "<labelvalue>",
+            ...
+          },
+            "annotations": {
+              "<labelname>": "<labelvalue>",
+            ...
+          },
+            "startsAt": "<rfc3339>", # optional, will be current time if not present
+            "endsAt": "<rfc3339>",
+            "generatorURL": "<generator_url>" # optional
+          },
+        ]
+        """
+
+        validSeverity = ["high", "medium", "low"]
```

Comment:
How about making it an object attribute in the init method (self.validSeverity)? Perhaps we could even have a method like:
```
def _isSeverityValid(self, severity):
    if severity not in validSeverity:
        logging.critical("Alert submitted to AlertManagerAPI with invalid severity: %s", severity)
        return False
    return True
```